app/views/mailers/notify_mailer/new_badge_email.html.erb

```diff
@@ -21,10 +21,10 @@
   </p>
   <hr style="opacity:0.4;width: 150px;margin: 30px auto;" />
   <p>
-    You also get <strong><a href="<%= ApplicationConfig["APP_DOMAIN"] %>/credits">5 new credits</a></strong> to use for <strong><a href="<%= ApplicationConfig["APP_DOMAIN"] %>/listings">community listings</a></strong><br />
+    You also get <strong><a href="https://dev.to/credits">5 new credits</a></strong> to use for <strong><a href="https://dev.to/listings">community listings</a></strong><br />
```

Comment:
I suppose that we don't want to use a hardcoded url here because in the future there could be other sites using the platform, the code should also work on staging. So the code should work without changing it on different domains.
I think `credit_url` will work here because `config.action_mailer.default_url_options` should be set. 

But, actually, we have `dev.to` over the `app/views/mailer` currently :D

Reply:
Yeah we should definitely avoid hardcode here. That's a great suggestion @lightalloy !

Try the following. 
```suggestion
    You also get <strong><a href="<%= credits_url %>">5 new credits</a></strong> to use for <strong><a href="<%= classified_listings_url %>">community listings</a></strong><br />
```